solution.js

```diff
@@ -0,0 +1,63 @@
+class Humans {
+    constructor (name, age){
+        this.name = name
+        this.age = age
+    }
+    
+ageOneyear(){
+    this.age = this.age + 1
+    return this.age
+}
+
+eating (){
+    return console.log("mmm, mmm, mmm, I'm love'n it")
+}
+}
+
+class Gender extends Humans{
+    constructor (name,age,gender)
+    {
+    super (name, age)
+    this.gender = gender
+    }
+}
+
+let kenny = new Humans ("Kenny", 28)
+let dough = new Humans("Dough", 18)
+
+console.log(kenny.age)
+
+console.log(kenny.ageOneyear())
+
+console.log(kenny.eating())
```

Comment:
Since your `eating` function is already logging the line, there's no need to console.log when calling with `kenny.eating()`. Let me know if this makes sense or if it's unclear what I'm talking about!